.changeset/blue-dots-battle.md

```diff
@@ -0,0 +1,27 @@
+---
+"@primer/react-brand": minor
+---
+
+Added `OrderedList` and `UnorderedList`
+
+Example Usage:
+
+```
+<UnorderedList variant="checked">
+    <li>Automatic security and version updates</li>
```

Comment:
This is using the <li> API?

Reply:
That is a mistake. Updating in next push